libraries/core-react/src/components/Typography/Typography.tsx

```diff
@@ -90,6 +91,14 @@ const StyledTypography = styled.p<StyledProps>`
         text-overflow: ellipsis;
       }
     `}
+  ${({ link }) =>
+    link &&
+    css`
+      width: max-content;
```

Comment:
Why are we setting `width`? Not exactly normal behaviour for an `a`element

Reply:
Width adjusted to the parent width, so focus was full width instead of wrapping the link itself

Reply:
This sounds like its something thats specific for when its used in `Breadcrumbs`? So maybe it should be applied there instead? We should avoid changing native behaviour unless its intended.

Reply:
@mimarz Yes grid was just missing `justify-items` so that fixed it 👍  Ready again for review now